### PR TITLE
Add pretty alignment and increase line length in scalafmt

### PR DESCRIPTION
**Pull request by tymm**

### CODE REVIEW

File: roscala/src/main/scala/coop/rchain/rosette/Monitor.scala [hunk 1]
```diff
@@ -6,9 +6,9 @@ case class Monitor(id: Ob,
                    obCounts: Long,
                    tracing: Boolean)
     extends Ob {
-  def reset(): Unit = {}
-  def start(): Unit = timer.start()
-  def stop(): Unit = timer.stop()
+  def reset(): Unit                  = {}
```
Comment (DoSOfRedRiver) [inline roscala/src/main/scala/coop/rchain/rosette/Monitor.scala]: Do we really want such alignment for methods?

File: roscala/src/main/scala/coop/rchain/rosette/Ob.scala [hunk 2]
```diff
@@ -210,50 +202,44 @@ object Ob {
...
+  def setLex(indirect: Boolean, level: Int, offset: Int, value: Ob): State[Ob, StoreResult] =
+    State { ob =>
+      val nthParentLens =
+        (0 until level).foldLeft(lens[Ob]: Lens[Ob, Ob])((l, _) => l >> 'parent)
+
+      def inSlotNum(lens: Lens[Ob, Ob]): Option[Lens[Ob, Ob]] =
+        if (!indirect) Some(lens)
         else {
-          val resLens = unsafeCastLens[Actor](lens) >> 'extension
-          Some(unsafeCastLens[Ob](resLens))
```
Comment (DoSOfRedRiver) [inline roscala/src/main/scala/coop/rchain/rosette/Ob.scala]: Is it a change from another PR?

**Comment by henrytill** [comment]
Looks great!

File: roscala/src/main/scala/coop/rchain/rosette/Monitor.scala (hunk 1 above)
Comment (rabbitonweb) [reply roscala/src/main/scala/coop/rchain/rosette/Monitor.scala]: @DoSOfRedRiver it really helps to skim through the code especially with tons of those small, tiny methods.

File: roscala/src/main/scala/coop/rchain/rosette/Ob.scala (hunk 2 above)
Comment (rabbitonweb) [reply roscala/src/main/scala/coop/rchain/rosette/Ob.scala]: I agree with @DoSOfRedRiver , this seem suspicious, almost as if leaked from different branch

Comment (tymm) [reply roscala/src/main/scala/coop/rchain/rosette/Ob.scala]: Hmm, things look fine when you just look at the file in Github.

Comment (kirkwood) [reply roscala/src/main/scala/coop/rchain/rosette/Ob.scala]: [Hiding whitespace changes](https://github.com/rchain/rchain/pull/234/files?w=1) shows that this is an artifact of the differ, not a real code change, maybe.

**Pull request merged**